tests/auth_basic_test.py

```diff
@@ -64,41 +65,63 @@ def test_signup_route(client, capture_templates, setup_logger):
         raise
 
 
-def test_signup_post_invalid_email(client, capture_templates, setup_logger):
+param_list = [
+    ("bad_email", "TestP@ssword!", HTTPStatus.OK),  # bad email
```

Comment:
The `HTTPStatus` is the same in all of these tuples, so I'd just delete that variable. If they varied at all, I'd keep it, but since the don't it's just a place go fumble your tests.

Reply:
updated!